apollo-federation/src/supergraph/mod.rs

```diff
@@ -713,16 +752,6 @@ fn extract_object_type_content(
             message: "@join__implements should exist for a fed2 supergraph".to_owned(),
         })?;
 
-    let context = supergraph_schema
```

Comment:
Refactors:
- Instead of refetching this supergraph schema spec each time, this has been moved into `validate_supergraph_for_query_planning()` (where other relevant supergraph schema specs are fetched).